daisy_integration_tests/image_export_vmdk_given_size.subwf.json

```diff
@@ -0,0 +1,38 @@
+{
+  "Name": "image-export-vmdk-given-size-test",
+  "Vars": {
+    "size": {
+      "Required": true
+    },
+    "gcs_export": {
+      "Value": "${SCRATCHPATH}/created-by-export-test-given-size-${size}gb.vmdk"
```

Comment:
You pass this var in in all the other workflows but you don't really need to, just have it in this workflow to reduce complexity, also just drop the size in the name, this path is fully scoped to this workflow by using SCRATCHPATH